sections/04_study.md

```diff
@@ -1,224 +1,65 @@
-appropriate features representing low/high pH, which can provide additional
-useful information and new features for future metagenomic sample comparison.
- Also, an initial study has show promise of these networks for diagnosing
-disease [Faruqi].  
-
-There are still a lot of challenges with applying deep neural networks to metagenomics problems.  They are not ideal for microbial/functional composition->phenotype classification because most studies contain tens of samples (~20->40) and
-hundreds/thousands of features (aka species).  Such underdetermined/ill-conditioned problems
-are still a challenge for deep neural networks that require many more training examples than 
-features to sufficiently converge the weights on the hidden layers.  Also, due to convergence issues 
-(slowness and instability due to large neural networks modeling very large datasets [@arxiv:1212.0901v2]), 
-taxonomic classification of reads from whole genome sequencing seems out of reach at the moment for deep neural 
-networks -- due to only thousands of full-sequenced genomes as compared to hundreds of thousands of 16S rRNA sequences 
-available for training.
-
-However, because recurrent neural networks are showing success for base-calling (and thus removing the large error in 
-the measurement of a pore's current signal) for the relatively new Oxford Nanopore sequencer [Boza], there is hope that
-the process of denoising->organism/function classification can be combined into one step in using powerful LSTM's.  LSTM's
-are working miracles in raw speech signal->meaning translation [ref], and combining steps in metagenomics are not out of the 
-question.  For example, metagenomic assembly usually requires binning then assembly, but could deep neural nets accomplish
-both tasks in one network?  Does functional/taxonomic classification need to be separate processes?  The largest potential
-in deep learning is to learn "everything" in one complex network, with a plethora of labeled (reference) data and unlabeled 
-(microbiome experiments) examples.
-
-### Sequencing and variant calling
-
-*We have one nanopore paper in the issues and very recent work on variant calling
-that looks worthy of inclusion.*
+Transcription Factor and RNA-binding proteins are key components for gene
+regulation, making them very important to understand for higher level
+biological processes. While high-throughput sequencing techniques such as
+Chromatin immunoprecipitation and massively parallel DNA
+sequencing (ChIP-seq) have been able to accurately identify binding regions for DNA and
+RNA proteins, these experiments are both time consuming and expensive. In
+addition, the sequencing methods do not provide any sort of analysis on the
+proteins which would lead to a better understanding of the underlying process.
+Thus, there is a need to computationally predict and understand these binding
+regions de novo from sequences.
+
+####Transcription Factors
```

Comment:
This line is causing the [codeclimate](https://codeclimate.com/github/greenelab/deep-review/pull/264#issues) error that

 > First header should be a top level header

This is the first header in the document. However, it's already at h4 (fourth level header, i.e. subsubheading).

In addition you're lacking a space between `####` and `Transcription Factors` causing:

> No space after hash on atx style header